### DISCUSSION TITLE
Cant create SSL Certs.

### DISCUSSION THREAD

**Discussion by severhart-fpcg** [rcl-ssl/docs#9] [General]
Error:Failed to create certificate with target Stand ALone for host : *.a******ital.com for challenge : DNS. Could not validate *.a****pital.com with challenge dns-01 for domain *.a*****tal.com, status reported : invalid. No TXT record found at _acme-challenge.a******tal.com . Recommendation : none

Keep getting this error when attempting to create a cert.. the TXT record is in place and dig does indeed see it as intended.

**Comment by anilripla** [comment]
What is the _acme-challenge end point? Is it '_acme-challenge.aital.com' ?  What domain are you trying to create TLS/SSL for?

**Comment by severhart-fpcg** [comment]
AnadarkoHospital.com

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Reply by anilripla** [reply]
can you try again and  post the unedited Error message?
